packages/flame/lib/src/test_helpers/random_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:math';
+import 'package:meta/meta.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+final _rnd = Random(); // for generating seeds
```

Comment:
If it is for seed generating I would personally give it a name that would reflect that usage.

Reply:
Sure, I'll do it.